src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.Face</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Face APIs.</Description>
-    <Version>2.3.0-preview</Version>
+    <Version>3.0.0-preview</Version>
```

Comment:
Is there a reason behind bumping major release version. Since this is preview, you can safely bump minor version number and release the nuget

Reply:
As I have explained in the PR description, we have added two new parameters in the API interface, so this will be a breaking SDK change. We have passed review and gotten exception for this from Azure API Review Board. In such case, do you still think bumping minor version will be good enough? Thanks.

Reply:
For a preview Nuget package breaking changes are acceptable.

Reply:
Sure, changed the version to `2.4.0-preview`.